src/player.cpp

```diff
@@ -41,6 +41,7 @@
 #include <bitset>
 
 #include <fstream>
+#include <cmath>
```

Comment:
OK, please do, and post when you push that so I can see it and grab this.  Thanks.